src/main/java/com/dongtronic/diabot/commands/NightscoutCommand.kt

```diff
@@ -130,6 +165,14 @@ class NightscoutCommand(category: Command.Category) : DiabotCommand() {
         builder.addField("mmol/L", mmolString, true)
         builder.addField("mg/dL", mgdlString, true)
         builder.addField("trend", trendString, true)
+        if (dto.iob > 0) {
+            builder.addField("iob", dto.iob.toString(), true)
+        } else if (dto.iob < 0) {
+            builder.addField("iob", dto.iob.toString(), true)
+        }
```

Comment:
Can be collapsed into `if (dto.iob != 0)`

Reply:
Should be fixed in newest commit needs to be if (dto.iob != 0.0F) but same logic